README.md

```diff
@@ -2,37 +2,41 @@
 
 This module handles opiniated VPC Service Controls and Access Context Manager configuration and deployments.
 
+
+## Compatibility
+This module is meant for use with Terraform 0.12. If you haven't [upgraded][terraform-0.12-upgrade] and need a Terraform 0.11.x-compatible version of this module, the last released version intended for Terraform 0.11.x
```

Comment:
Add link to the [terraform-0.12-upgrade] pointer at the bottom. Like:
`[terraform-0.12-upgrade]: https://www.terraform.io/upgrade-guides/0-12.html`

Reply:
Thank you, Pavlo
[Resolved](https://github.com/terraform-google-modules/terraform-google-vpc-service-controls/commit/63c5a8a98dbafd8436547a6d994293cafb99c825)